apps/web-giddh/src/app/shared/template-froala/template-froala.component.html

```diff
@@ -4,54 +4,90 @@
     (commonLocaleData)="commonLocaleData = $event"
     (localeData)="localeData = $event"
 >
-    <div class="dialog-header d-flex align-items-center form-header">
+    <div class="dialog-header d-flex align-items-center form-header position-sticky top-0">
         <h3>{{ commonLocaleData?.app_custom_email_template }}</h3>
         <span mat-dialog-close aria-label="close icon"><i class="icon-cross"></i></span>
     </div>
+
     <mat-dialog-content class="dialog-body d-flex flex-column h-100">
         <div class="overflow-auto">
             <form [formGroup]="emailForm" class="d-flex flex-column flex-grow-1">
-                <div class="chip-gap">
-                    <div class="form-group position-relative">
-                        <select-multiple-fields
-                            [placeholder]="localeData?.select_to"
-                            [options]="toEmails"
-                            (selectedOption)="selectedToEmails = $event"
-                            [selectedValues]="selectedToEmails"
-                            [validations]="['email']"
-                            [name]="'to'"
-                            class="select-multiple-chiplist mb-3"
-                        ></select-multiple-fields>
-                        <div class="d-flex column-gap1 position-absolute chiplist-section">
-                            <a class="btn-link under" href="javascript:;" (click)="toggleBccCc('cc')">{{
-                                localeData?.cc
-                            }}</a>
-                            <a class="btn-link" href="javascript:;" (click)="toggleBccCc('bcc')">{{
-                                localeData?.bcc
-                            }}</a>
+                <div class="chip-gap" (clickOutside)="clickedOutsideEmail()" (click)="clickedInsideEmail()">
+                    <div class="form-group position-relative" (click)="clickedInsideEmailSection && setEmailFocus(emailType.To)">
+                        <div class="d-flex border-bottom">
+                            <label *ngIf="clickedInsideEmailSection" class="mr-r1">{{localeData?.to}}</label>
+                            <div [hidden]="!allStaticEmails.length" class="w-100 mr-b05">
+                                {{allStaticEmails}}
+                                <span class="suffix-emails font-15 d-inline-block text-primary" *ngIf="isHiddenEmails.length" (click)="$event.stopPropagation(); clickedInsideEmail()">{{isHiddenEmails}}</span>
```

Comment:
what's the use of stopPropagation here?

Reply:
When the user clicks on the hide email section, both the inside and outside functions are triggered. To prevent this, I used stopPropagation and called only the inside function.